Localization/fr snagit

theeagle opened the pull request.

theeagle committed:
stop referring to generic localized string macro

causes another localizable.strings file to be created and this causes issues for lots of things

theeagle committed:
Remove the rogue localizable file

aprilnoren commented on PTHotKey/PTKeyCombo.m:
```diff
@@ -264,7 +264,7 @@ - (NSString*)description
 	else
 	{
-		desc = NSLocalizedString( @"(None)", @"Hot Keys: Key Combo text for 'empty' combo" );
+      desc = @"(None)";
```
Does this allow it to use the localization that is already setup?

theeagle replied:
My logic was this is solely for debugging purposes (it's in the ```description``` method, and removing it let me get rid of another localizable.strings file.

theeagle committed:
Remove a bunch of unused strings files

I kept the languages we're using plus a couple the execs have talked about adding in the near-ish future.
